### DISCUSSION TITLE
Installation of qudi-iqo-modules

### DISCUSSION THREAD

**Discussion by qku** [Ulm-IQO/qudi-core#26] [Q&A] [answered]
Hello,
I am trying to start working with the new qudi-core. At the same time, I would like to use some of the modules contained in the qudi-iqo-modules collection. What is the preferred way to install those after installing qudi-core via pip? I could not find any documentation on this and would therefore greatly appreciate assistance!
Thank you :)

**Comment by timoML** [comment, marked as answer]
It's nice that you started moving to the new core! We're working hard towards the release, currently. The new core is not officially supported yet. However, we highly appreciate the feedback of everyone who is already migrating.
I would point you at the qudi-iqo-modules [readme](https://github.com/Ulm-IQO/qudi-iqo-modules), [installation instructions](https://github.com/Ulm-IQO/qudi-iqo-modules/blob/main/docs/installation_guide.md) and [porting guide](https://github.com/Ulm-IQO/qudi-iqo-modules/blob/main/docs/migrating_from_qudi_v0.1.md). Feel free to ask here, if there is anything unclear. This would also help a lot in getting the iqo-modules documentation ready for a release.

edit: I changed the links to the updated documentation that is now available in qudi-iqo-modules main branch.
